parm/config/config.fv3

```diff
@@ -87,14 +87,14 @@ case $case_in in
         export WRTIOBUF="8M"
         ;;
     "C384")
-        export DELTIM=${DELTIM:-240}
+        export DELTIM=${DELTIM:-300}
         export layout_x=6
         export layout_y=8
-        export layout_x_gfs=${layout_x_gfs:-6}
-        export layout_y_gfs=${layout_y_gfs:-8}
+        export layout_x_gfs=${layout_x_gfs:-8}
+        export layout_y_gfs=${layout_y_gfs:-12}
         export nth_fv3=2
         export nth_fv3_gfs=${nth_fv3_gfs:-2}
-        export cdmbgwd="1.0,2.2,1.0,1.0"  # mountain blocking, ogwd, cgwd, cgwd src scaling
+        export cdmbgwd="1.1,0.72,1.0,1.0"  # mountain blocking, ogwd, cgwd, cgwd src scaling
```

Comment:
Have these updates been testing in atmos-only mode when cycling with the enkf? This may have a significant impact on the C384 enkf forecast jobs, particularly the layout which will change the calculated resources.

Reply:
I have not run any cycled tests.